globalpayments-gateway-provider-for-woocommerce.php

```diff
@@ -7,14 +7,48 @@
  * Requires PHP: 7.1
  * WC tested up to: 7.2.2
  * Author: Global Payments
-*/
+ */
 
 defined( 'ABSPATH' ) || exit;
 
 if ( version_compare( PHP_VERSION, '7.1', '<' ) ) {
+	add_action( 'admin_notices', function () {
+		$message = sprintf( __( 'Your PHP version is %s but GlobalPayments For WooCommerce requires version 7.1+.', 'globalpayments-gateway-provider-for-woocommerce' ), PHP_VERSION );
+		echo '<div class="notice notice-error"><p>' . $message . '</p></div>';
+	} );
+
 	return;
 }
 
+register_activation_hook( __FILE__, 'plugin_activation' );
+
+function plugin_activation() {
```

Comment:
Please prefix the function name with "globalpayments_" to avoid name collision with other plugins.